database/prisma.yml

```diff
@@ -1,11 +1,5 @@
-# The service name, this will be part of the endpoint of the Prisma API
-service: hackernews-node
-
-# The deployment stage, this will also be part of the API endpoint
-stage: dev
-
-# Points to the file that holds your data model
+endpoint: __PRISMA_ENDPOINT__
 datamodel: datamodel.graphql
-
-# You can only access the API when providing JWTs that are signed with this secret
-secret: mysecret123
+hooks:
+  post-deploy:
+    - graphql get-schema --project database
```

Comment:
Ah, I also just realized that we probably want to add the GraphQL CLI as a dev dependency to the project as well so this won't fail when people just want to check out the repo.

Reply:
btw, if it's added as a dev dependency we also need to add it as a _script_ to `package.json` and then have the `post-deploy` hook be `yarn graphql get-schema --project database` to invoke the script. I'd also add a comment that quickly highlights the fact that in the repo the `graphql-cli` is just installed as a dev dependency so that's why we're prefixing it with `yarn`.